fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rewrite/ScalarRangePredicateExtractor.java

```diff
@@ -88,7 +88,7 @@ private ScalarOperator rewrite(ScalarOperator predicate, boolean onlyExtractColu
             }
         }
 
-        if (extractExpr != null && !conjuncts.contains(extractExpr)) {
+        if (!conjuncts.contains(extractExpr)) {
```

Comment:
The necessary? extractExpr could always be non-null？

Reply:
It's been checked before, don't need check again